service-level-sidecar/scuba.js

```diff
@@ -0,0 +1,80 @@
+const arsenal = require('arsenal');
+const bucketclient = require('bucketclient');
+
+const { BucketClientInterface } = arsenal.storage.metadata.bucketclient;
+const { splitter } = arsenal.constants;
+
+const rootLogger = require('./log');
+const env = require('./env');
+const utils = require('./utils');
+
+const params = {
+    bucketdBootstrap: [env.scubaBucketd],
```

Comment:
the readme says `bucketd bootstrap list`. If it's a list should it be provided like `abc:19000,def:19000` ?
```suggestion
    bucketdBootstrap: env.scubaBucketd.split(','),
```

Reply:
It should just be a single value. I've changed the wording in the readme.